### PR TITLE
refactor: use orgsLimit instead of orgsEnabled as org quota key

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

use orgsLimit instead of orgsEnabled as org quota key.

## Testing

N/A

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by simeng-li — APPROVED** [review]
core updates LGTM

File: packages/console/src/components/PlanUsage/index.tsx [hunk 1]
```diff
@@ -59,6 +59,9 @@ function PlanUsage({ periodicUsage: rawPeriodicUsage }: Props) {
       usage:
         key === 'mauLimit' || key === 'tokenLimit'
           ? periodicUsage[key]
```
Comment (charIeszhao) [inline packages/console/src/components/PlanUsage/index.tsx:61]: This expression is too complicated. Need to split into several smaller pieces

File: packages/console/src/pages/Organizations/index.tsx [hunk 2]
```diff
@@ -35,7 +36,9 @@ function Organizations() {
   const [isCreating, setIsCreating] = useState(false);
 
   const isOrganizationsDisabled =
-    isCloud && !currentSubscriptionQuota.organizationsEnabled && planId !== ReservedPlanId.Pro;
+    isCloud &&
+    !isFeatureEnabled(currentSubscriptionQuota.organizationsLimit) &&
+    planId !== ReservedPlanId.Pro;
```
Comment (charIeszhao) [inline packages/console/src/pages/Organizations/index.tsx:41]: What if the planId === Pro?

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/components/PlanUsage/index.tsx (hunk 1 above)
Comment (darcyYe) [reply packages/console/src/components/PlanUsage/index.tsx]: Refactored.

**Review by wangsijie — APPROVED** [review]

**Pull request merged**